drupal-11.advisories.yaml

```diff
@@ -9,6 +9,11 @@ advisories:
       - CVE-2012-2306
       - GHSA-g4rv-c45g-4j68
     events:
+      - timestamp: 2025-06-23T19:40:33Z
+        type: false-positive-determination
+        data:
+          type: vulnerable-code-version-not-used
+          note: This vulnerability is a false positive and not present in recent versions of Drupal.
```

Comment:
Can we include some evidence / reasoning to back up that claim? can he high level, but usually if it was confirmed by upstream maintainers, we'd like a GH issue, or write up our own why / how